python/nav/web/templates/maintenance/frag-component-trail.html

```diff
@@ -0,0 +1,11 @@
+{% load maintenance %}
+{% for element in trail %}
+    {% with element|component_name as title %}
+        {% if element.get_absolute_url %}
+            <a href="{{ element.get_absolute_url }}" title="{{ title }}">{{ title }}</a>
```

Comment:
actually, the link title here should be different from the link text (i.e. description for rooms and locations, ip address for netboxes)

Reply:
Fixed.